backend/typescript/seeders/20211114043430-seed-donations.ts

```diff
@@ -33,11 +33,13 @@ const seedSchedules = [
     categories: ["Dry packaged goods"],
     size: "medium",
     is_pickup: false,
+    day_part: "Morning (6am - 11am)",
     start_time: new Date("2021-09-30T00:00:00.000Z"),
     end_time: new Date("2021-10-01T00:00:00.000Z"),
     status: "Approved",
     volunteer_needed: true,
-    frequency: "One-time",
+    volunteerTime: "8:00 AM",
```

Comment:
this should be volunteer_time, cuz this is being added straight to the table, not through the endpoint. Same with the other seeder volunteerTime

Reply:
Thank you for catching this!